core/src/main/java/org/apache/calcite/sql/parser/StringAndPos.java

```diff
@@ -28,6 +31,39 @@ public class StringAndPos {
   public final String sql;
   public final int cursor;
   public final @Nullable SqlParserPos pos;
+  private static final Set<String> BITXOR_OPERATOR_WHITELISTED_SQLS;
+
+  static {
```

Comment:
are these test cases?
they do not belong to this class; they should in a test package